app/gui/view/debug_scene/cloud-dashboard/cloud-http/src/model.rs

```diff
@@ -0,0 +1,235 @@
+            /// is [`response::project::$list_projects`].
+            ///
+            /// [`Project`]: ::enso_cloud_view::project::Project
+            #[display(fmt = "/projects")]
+            $list_projects,
+            /// Endpoint for opening a [`Project`] that is not currently open. The response type
+            /// for this endpoint is [`response::project::$open_project`].
+            ///
+            /// [`Project`]: ::enso_cloud_view::project::Project
+            #[display(fmt = "/projects/{}/open", _0)]
+            $open_project(view::id::ProjectId),
+            /// Endpoint for closing a [`Project`] that is not currently closed. The response type
+            /// for this endpoint is [`response::project::$close_project`].
+            ///
+            /// [`Project`]: ::enso_cloud_view::project::Project
+            #[display(fmt = "/projects/{}/close", _0)]
+            $close_project(view::id::ProjectId),
+        }
+
+
+        // === Main `impl` ===
+
+        impl Route {
+            /// Returns the [`http::Method`] used to interact with this route.
+            ///
+            /// [`http::Method`]: ::http::Method
+            pub fn method(&self) -> http::Method {
+                match self {
+                    Self::$list_projects => http::Method::GET,
+                    Self::$open_project(_) => http::Method::POST,
+                    Self::$close_project(_) => http::Method::POST,
+                }
+            }
+        }
+
+
+
+        // ===============
+        // === Request ===
+        // ===============
+
+        /// Module containing request types for the Cloud API.
+        ///
+        /// See [`crate::model`] for more details.
+        pub mod request {
+
+
+
+            // ===============
+            // === Project ===
+            // ===============
+
+            /// Module containing request types for [`Project`]-related [`Route`]s.
+            ///
+            /// [`Route`]: crate::Route
+            /// [`Project`]: ::enso_cloud_view::project::Project
+            pub mod project {
+
+
+
+                // ===================
+                // === OpenProject ===
+                // ===================
+
+                /// A request to the [`OpenProject`] [`Route`], which starts the process of opening
+                /// (i.e., start running) a [`Project`] that is not currently [`Opened`].
+                ///
+                /// [`OpenProject`]: crate::Route::$open_project
+                /// [`Route`]: crate::Route
+                /// [`Project`]: ::enso_cloud_view::project::Project
+                /// [`Opened`]: ::enso_cloud_view::project::StateTag::Opened
+                #[derive(Clone, Copy, Debug, serde::Deserialize, serde::Serialize)]
+                #[allow(missing_docs)]
+                pub struct $open_project {
+                    /// Whether the Cloud instance (i.e., backing VM) that the [`Project`] runs on
```

Comment:
```suggestion
                    /// Whether the Cloud instance (i.e. backing VM) that the [`Project`] runs on
```

Reply:
(North) American English vs. British English 😉 

We may have the ~Queen~ King on our money, but we don't follow the same stylistic rules.